openmdao/docs/basic_guide/basic_case_recording.rst

```diff
@@ -0,0 +1,41 @@
+.. _basic_case_recording:
+
+************************
+Basic Recording Example
+************************
+
+Recording Terminology
+---------------------
+
+| **Case**: A case is an OpenMDAO model.
```

Comment:
A Case is actually a snapshot of the _state_ of a model..  or a subset of that state.